cmp/cmpopts/sort.go

```diff
@@ -94,7 +94,8 @@ func (ss sliceSorter) less(v reflect.Value, i, j int) bool {
 //	• Transitive: if !less(x, y) and !less(y, z), then !less(x, z)
 //	• Total: if x != y, then either less(x, y) or less(y, x)
 //
-// SortMaps can be used in conjuction with EquateEmpty.
+// SortMaps can be used in conjunction with EquateEmpty,
+// but cannot be used with DiscardMapZeros.
```

Comment:
Hmm, this is an interesting data point for our "only one option may apply" heuristic. These options happen to be commutative, so it's safe (and not particularly confusing) to apply them both.

Perhaps this is another good use-case for the "first option that applies" combiner you mentioned at GopherCon.